### ISSUE TITLE
Add SILVA clusters

### ISSUE THREAD

**Issue by jar398** [OpenTreeOfLife/reference-taxonomy#123]

**Comment by jar398** [comment]
Just reviewing long-standing issues...
(a) I am sort of waiting for a blessing from Karen/Mark/Stephen before proceeding. They may be waiting for more details on how this will work.
(b) It doesn't make a lot of sense to do this without some way in OTU mapping and/or the TNRS to handle genbank ids. So that work needs to be spec'd out and done.

**Comment by kcranston** [comment]
Agreed that this needs discussion. I am not yet convinced that adding the SILVA clusters will improve the tree enough for this to rise to the top of the To Do list. Seems like there are many other projects of higher priority.
